src/test/functional/features/sole-conditional-order.feature

```diff
@@ -49,3 +49,38 @@ Feature: Sole conditional order
     And I click "I believe that the facts stated in this application are true"
     And I click "Continue"
     Then the page URL should be "/hub-page"
+
+  Scenario: Applicant 1 response with additional information for condition order
+    And I set the case state to "AwaitingClarification"
+    When I go to "/provide-information-to-the-court"
+    Then the page should include "Upload any documents"
+
+    When I clear the form
+    Given I select "If the court wants you to explain something or provide additional information then write your response here. If the court has just asked you to upload documents then you do not have to write anything, unless you think it’s useful information."
+    And I type "test details"
+    And I select "I cannot upload some or all of my documents"
+    Then the page should include "You can post your documents to the court"
+
+    When I click "Continue"
+    Then the page URL should be "/hub-page"
+
+  Scenario: Applicant 1 response with additional information and uploads for condition order
+    And I set the case state to "AwaitingClarification"
+    When I go to "/provide-information-to-the-court"
+    Then the page should include "Upload any documents"
+
+    When I clear the form
+    And I select "If the court wants you to explain something or provide additional information then write your response here. If the court has just asked you to upload documents then you do not have to write anything, unless you think it’s useful information."
+    And I type "test details"
+
+    Given I delete any previously uploaded files
+    Then the page should include "No files uploaded"
+    When I upload the file "fixtures/larry-the-cat.jpg"
+    Then I wait until the page contains image "larry-the-cat.jpg"
+    And I click "Delete"
+    And I wait until the page doesn't contain "larry-the-cat.jpg"
+    And I select "I cannot upload some or all of my documents"
+    Then the page should include "You can post your documents to the court"
+
+    When I click "Continue"
+    Then the page URL should be "/hub-page"
```

Comment:
Can this just be one test? - considering our tests were failing before because we had so many

Reply:
sure, my idea was i was going to add flaky to this as one of them is an upload test.

Reply:
ah yeah, happy if you want to do that then

Reply:
will do just want to see if it passes before i add the tag